responsibleai/responsibleai/modelanalysis/model_analysis.py

```diff
@@ -216,6 +225,14 @@ def _validate_model_analysis_input_parameters(
                     'The serializer should be serializable via pickle')
 
         if isinstance(train, pd.DataFrame) and isinstance(test, pd.DataFrame):
+            if test.shape[0] > maximum_rows_for_test:
+                msg_fmt = 'The test data has {0} rows, ' +\
```

Comment:
can you make the message more actionable, specifically by stating something like:
"Please downsample your dataset to have fewer than 5000 rows or increase maximum_rows_for_test parameter value"

Reply:
Done.